### PR TITLE
Add base thing to remember to the Websites, Stores, and Views page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request adds base thing to remember to the Websites, Stores, and Views page

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/stores/websites-stores-views.html

### CODE REVIEW

File: src/stores/websites-stores-views.md [hunk 1]
```diff
@@ -16,3 +16,17 @@ Every Adobe Commerce and Magento Open Source installation has a [hierarchy]({% l
...
+
+1. Every Magento website has at least one default store and store view.
+
+1. You can set a different base URL for each store view.
+
+1. The main purpose of the website is to configure the features.
+
+1. The main purpose of the store is to configure the root category.
+
+1. The main purpose of the store view is to set translation information and currency symbols.
```
Comment (leslietilling) [inline src/stores/websites-stores-views.md:32]:
```suggestion
When working with websites, stores, and store views, keep the following in mind:

- Commerce instances have a cascading model: global → website → store → store view.
- Every website has at least one default store and store view.
- Each store view can have a different base URL.
- The primary function of a website is top-level feature configuration.
- The primary function of a store is root category configuration.
- The primary function of a store view is translation information and currency symbol configuration.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Comment by ZhovnirMaria** [comment]
@leslietilling hi! Please check, thanks ☺️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**